#167290437 Clear out warring report by test

blackshady opened the pull request:
#### What does this PR do? 
 Eradicate the warrings showing up while running test.

#### Description of Task to be completed?

- [x] Clear out warnings form the console
- [x]  Add some test case

#### How should this be manually tested?

1.  Pull this branch `git pull ch-fix-warning-report-167290437`
2.  Run `yarn or yarn install`
3.  After you dependencies are done installing run `yarn test`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#167290437](https://www.pivotaltracker.com/story/show/167290437)

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
<img width="1023" alt="Screenshot 2019-07-17 at 3 39 29 PM" src="https://user-images.githubusercontent.com/26234445/61434407-ca0d4300-a92d-11e9-8e1f-cc5e5cfe7522.png">

bozicschucky reviewed (approved):
I have tested the branch locally and everything seems to work as expected, good work @blackshady

09arnold reviewed (commented):
Looks good, but there's some merge conflicts and code climate issues you should look into

akhilome reviewed (approved):
This looks good to go. You do have multiple commits, though.

ssewilliam commented on src/components/developerCards/index.jsx:
```diff
@@ -83,26 +82,13 @@ class DeveloperCard extends Component {
       const newName = _.split(name, ',');
       const firstInitials = newName[0].charAt(0);
-      const secondInitials = newName[1] && newName[1].charAt(1);
+      const secondInitials = newName[1].charAt(1);
```
@blackshady   This change is likely to return the bug that was solved [here](https://www.pivotaltracker.com/story/show/167191688) and fixed through this [PR](https://github.com/andela/bp-esa-frontend/pull/101)

blackshady replied:
Hi @ssewilliam this was coming from a different commit. it has been resolved.

ssewilliam commented on src/components/developerCards/index.jsx:
```diff
@@ -83,26 +82,13 @@ class DeveloperCard extends Component {
-              <img className="info-icon" src={InfoIcon} alt="info icon" role="presentation" id={`${index}-id`} />
-              <span className="tooltiptext">Details</span>
-            </div>
+            { /* TODO Update the onboarding icon to match mockups */}
```
@blackshady were you expecting to do this in this PR and forgot about it or I should create a story about it.

ssewilliam commented on src/components/ReportPage/index.jsx:
```diff
@@ -459,8 +456,8 @@ export const mapDispatchToProps = dispatch => ({
-  fetchStat: period => dispatch(fetchStatsRequest(period)),
-  retryFailedAutomation: automationId => dispatch(retryAutomation(automationId)),
+  fetchStat: (period) => dispatch(fetchStatsRequest(period)),
+  retryFailedAutomation: () => dispatch(retryAutomation()),
```
Hey, @blackshady  retryAutomations expects AutomationId but you have taken it out. Is this intended?

https://github.com/andela/bp-esa-frontend/blob/28fb2fc768824f0bd129ef380e008548153da6b1/src/redux/actions/automationActions.js#L27

blackshady committed:
chore(test): fix warring report on test
 - Clear out warnings form the console
 - Add some test case

 [Finishes 167290437]

ssewilliam reviewed (commented):
Hey @blackshady please resolve the merge conflicts

blackshady commented:
Hi @ssewilliam the merge conflict has been fixed
